CareKit/CareCard/OCKCareCardViewController.h

```diff
@@ -57,6 +57,18 @@ NS_ASSUME_NONNULL_BEGIN
 - (BOOL)careCardViewController:(OCKCareCardViewController *)viewController shouldHandleEventCompletionForActivity:(OCKCarePlanActivity *)interventionActivity;
 
 /**
+ Asks the delegate to verify that the given activity can be discontinued [deleted]. If this method is not implemented, all
+ all rows are assumed to not be discontinuable.
+ 
+ If returned YES, and the user chooses to discontinue the activity, the activity's end date will be set to the date of the
+ event that was selected, thereby removing all subsequent events.
+ 
+ @param viewController              The view controller providing the callback.
+ @param interventionActivity        The intervention activity that the user selected.
+ */
```

Comment:
Update documentation to state that the discontinue will happen after "today's" date. And this is only applicable to the current day. If the user goes back to a previous day, they cannot mark it to be discontinued.